<launch>
<group ns="/robot1/map_merge">
  <param name="init_pose_x" value="10.5"/>
  <param name="init_pose_y" value="-4.5"/>
  <param name="init_pose_z" value="0"/>
  <param name="init_pose_yaw" value="0.0"/>
</group>
<group ns="/robot2/map_merge">
  <param name="init_pose_x" value="9"/>
  <param name="init_pose_y" value="-1"/>
  <param name="init_pose_z" value="0.0"/>
  <param name="init_pose_yaw" value="0.0"/>
</group>
<group ns="/robot3/map_merge">
  <param name="init_pose_x" value="7.3"/>
  <param name="init_pose_y" value="4.5"/>
  <param name="init_pose_z" value="0.0"/>
  <param name="init_pose_yaw" value="0.0"/>
</group>
<group ns="/robot4/map_merge">
  <param name="init_pose_x" value="6.7"/>
  <param name="init_pose_y" value="7.3"/>
  <param name="init_pose_z" value="0.0"/>
  <param name="init_pose_yaw" value="0.0"/>
</group>

<group ns="map_merge">
  <node pkg="multirobot_map_merge" type="map_merge" respawn="false" name="map_merge" output="screen">
    <param name="robot_map_topic" value="map"/>
    <param name="robot_namespace" value=""/>
    <param name="merged_map_topic" value="map"/>
    <param name="world_frame" value="world"/>
    <param name="known_init_poses" value="false"/>
    <param name="merging_rate" value="0.5"/>
    <param name="discovery_rate" value="0.05"/>
    <param name="estimation_rate" value="0.1"/>
    <param name="estimation_confidence" value="1.0"/>
  </node>
</group>


</launch>
